Main.java

```diff
@@ -1,6 +1,8 @@
+import java.util.Scanner;
+
 /**
- *
- * @author 
+ * This program checks if an integer is even or odd
+ * @chad

```

Comment:
The @author is a special tag and should actually stay there. It is something called javadoc. So instead, just have this line be @author chad.